src/background/RemotePontoon.ts

```diff
@@ -116,21 +116,27 @@ async function updateNotificationsIfThereAreNew(pageContent: string) {
   }
 }
 
-async function updateNotificationsData() {
+export async function updateNotificationsData() {
```

Comment:
```suggestion
async function updateNotificationsData() {
```

No need for export, unless this gets directly called from somewhere, which is not the case (yet?).